ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/apitesting/CommandTransactionChecks.scala

```diff
@@ -220,22 +209,24 @@ class CommandTransactionChecks(
         ) // expected args
       }
 
-      // TODO An equivalent of this in TransactionIT tests this with a lower number of commands.
-      // This can be removed once we can raise the command limit there.
-      "accept huge submissions with a large number of commands" in {
+      /*
+       * TODO(FM) for absolutely mysterious reasons this times out, but the equivalent one in TransactionServiceIT
```

Comment:
@gaboraranyossy-da do you have an idea on why this might happen?

Reply:
debugging it locally it seems that this call doesn't return: https://github.com/digital-asset/daml/blob/05e691f55852fbb207f0e43cf23bb95b95866ba3/ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/CommandExecutorImpl.scala#L31 It probably dies on the ExecutionContext.